ui/E2E.md

```diff
@@ -0,0 +1,47 @@
+# UI Tests
+
+The ui app contains a suite of automated e2e tests that run in CI and can be run locally.
+
+The main test script is located in `rube/index.ts` and uses [Playwright](https://playwright.dev) to run.
+
+The script:
+
+1. fetches fake ships piers from GCS
+2. downlaods the urbit binaries for the local architecture
+3. boots the ships
+4. executes the tests against the current ui code version
+5. kills spawned processes
+
+Test specs are in the `e2e/` directory.
+
+## Running locally
+
+After installing  this project's dependencies with `npm install`,
+
+```
+npx playwright install
+```
+
+Then, to run all the tests:
+
+```
+npm run rube
```

Comment:
```suggestion
npm run e2e
```

We're aliasing it to e2e in case we need to change the underlying script(s) later.